### PR TITLE
feat: V4 download pay activity CSV [3/n]

### PR DESCRIPTION
**Pull request by johnnyd-eth**
Screenshot 2024-07-25 at 2 58 59 pm

Screenshot 2024-07-25 at 2 59 20 pm

Screenshot 2024-07-25 at 3 01 36 pm

### CODE REVIEW

File: src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import { DownloadOutlined } from '@ant-design/icons'
```
Comment (aeolianeth) [inline src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx:1]: Let's use a heroicons equivalent

Screenshot 2024-07-28 at 7 49 39 AM

File: src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import { t, Trans } from '@lingui/macro'
+import { Button, Modal, ModalProps } from 'antd'
+import InputAccessoryButton from 'components/buttons/InputAccessoryButton'
+import FormattedNumberInput from 'components/inputs/FormattedNumberInput'
+import { useBlockNumber } from 'hooks/useBlockNumber'
```
Comment (aeolianeth) [inline src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx:6]: Lets use the wagmi hook for this: https://wagmi.sh/react/api/hooks/useBlockNumber#useblocknumber

**Review by aeolianeth — APPROVED** [review]

File: src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx (hunk 2 above)
Comment (johnnyd-eth) [reply src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx:6]: Done

File: src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx (hunk 1 above)
Comment (johnnyd-eth) [reply src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4ActivityPanel/DownloadActivityModal.tsx:1]: Good pickup, thanks

**Pull request merged**